Assets/Scripts/System/TypeScriptGenerationSystem.cs

```diff
@@ -794,6 +794,15 @@ private string GenerateActualScript(GenerationInfo generationInfo)
 
             generatedScript.AppendLine("}".Indent(0));
 
+            var lines = generatedScript.ToString().Split("\n");
+            generatedScript.Clear();
+            for (int i = 0; i < lines.Length; i++)
+            {
+                if (i % 30 == 0)
```

Comment:
With the current solution, a situation like this can happen:
![grafik](https://github.com/metagamehub/dcl-edit/assets/11379989/48d12f02-e9d7-47a8-abc1-b0096eaee5b7)
This leads to a wrong inline documentation:
![grafik](https://github.com/metagamehub/dcl-edit/assets/11379989/98d4a20b-2b27-46ea-ac40-9aa890f16309)